packages/bvaughn-architecture-demo/components/console/renderers/shared.module.css

```diff
@@ -172,3 +195,27 @@
 .ErrorBoundaryFallback {
   color: var(--color-error);
 }
+
+:global(.theme-light) .ColorBadge {
+  --pulse-color-1: rgba(127, 127, 127, 0.1);
+  --pulse-color-2: rgba(97, 97, 97, 0);
+  --pulse-color-3: rgba(88, 88, 88, 0);
+}
+
+:global(.theme-dark) .ColorBadge {
+  --pulse-color-1: rgba(185, 185, 185, 0.35);
+  --pulse-color-2: rgba(204, 169, 44, 0);
+  --pulse-color-3: rgba(204, 169, 44, 0);
+}
+
+@keyframes pulse {
+  0% {
+    box-shadow: 0 0 0 0 var(--pulse-color-1);
+  }
+  70% {
+    box-shadow: 0 0 0 0.45rem var(--pulse-color-2);
```

Comment:
I'm curious about `.45rem` vs `.5rem` ? It's fine, but I think we general have been going in increments of `.25rem` in other places, so I just wanted to ask to make sure we had a good reason to go with an unusual one here 😁